route/v1/types.go

```diff
@@ -350,6 +352,8 @@ type RouteStatus struct {
 	// ingress describes the places where the route may be exposed. The list of
 	// ingress points may contain duplicate Host or RouterName values. Routes
 	// are considered live once they are `Ready`
+	// +listType=map
+	// +listMapKey=name
```

Comment:
Should this be routerName?

Reply:
updated to `routerName`. However, is it better to get ack from  NE team as well?

Reply:
I've already DM'd @Miciah to ask for a second opinion